_compress_images_cache.yml

```diff
@@ -1795,3 +1795,32 @@ assets/images/home/banner/ongoing-development-mobile.png: ongoing-development-mo
 assets/images/home/banner/ongoing-development-mobile@2x.png: ongoing-development-mobile@2x.png
 assets/images/home/banner/ongoing-development.png: ongoing-development.png
 assets/images/home/banner/ongoing-development@2x.png: ongoing-development@2x.png
+assets/images/guide/getting-started/principles/principle-security.png: principle-security.png
```

Comment:
I accidentally re-compressed my images when shifting folders too, isn't an issue rn but if we keep doing it they may become lower resolution - just something to keep in mind!

Reply:
I left these lines in the compress images cache to prevent them from being re-compressed. The jekyll compiler should ignore them now that it sees these images as already compressed.